packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js

```diff
@@ -177,14 +188,15 @@ async function transformerVersionCheck(context, resourceDir, cloudBackendDirecto
       if (!response.transformerConfig) {
         process.exit(0);
       }
+      transformerVersion += 1;
     }
   }
 
   // Only touch the file if it misses the Version property
   // Always set to the base version, to not to break existing projects when coming
   // from an older version of the CLI.
-  if (!localTransformerConfig.Version) {
-    localTransformerConfig.Version = TRANSFORM_BASE_VERSION;
+  if (!localTransformerConfig.Version || transformerVersion) {
+    localTransformerConfig.Version = transformerVersion || TRANSFORM_BASE_VERSION;
```

Comment:
TRANSFORM_CURRENT_VERSION instead of base version?

Reply:
The questionflow of adding the transformer version would always be `transformerVersion`. If we change the base to current we would not want to bump a customer to the current version should the transformer version not be available. Though double checking this I'm not sure if that the base version is needed at all.